### ISSUE TITLE
Plans to continue?

### ISSUE THREAD

**Issue by ConDotBones123** [TomjWolcott/arbgeom_rs#1]
Are there any plans to update the Arbgeom game further?

**Comment by TomjWolcott** [comment]
Yes!  I've actually been able to work on it a bit more lately.  Right now I'm finishing up a minimal version of the scripting for the game before I shift my focus back to the physics (I had to change my approach to scripting from what I was doing earlier due to performance issues).  If you want to see, I have the scripting language I made for it [here](https://github.com/TomjWolcott/scaffold_scripting).  Right now, the scripting is a mix of the scaffold language to define the projection/sdf/aabb functions, ron the combine objects and define levels, and rhai to do whatever else.  

I do want to upload an updated version to itch.io soon, hopefully I can improve the physics quickly enough to include those improvements in the update.
